docs/copied-from-beats/dashboards.asciidoc

```diff
@@ -0,0 +1,83 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/dashboards.asciidoc[]
+//////////////////////////////////////////////////////////////////////////
+
+
+{beatname_uc} comes packaged with example Kibana dashboards, visualizations,
+and searches for visualizing {beatname_uc} data in Kibana. Before you can use
+the dashboards, you need to create the index  pattern, +{beat_default_index_prefix}-*+, and
+load the dashboards into Kibana. To do this, you can either run the `setup`
+command (as described here) or
+<<configuration-dashboards,configure dashboard loading>> in the
++{beatname_lc}.yml+ config file.
+
+NOTE: Starting with {beatname_uc} 6.0.0, the dashboards are loaded via the Kibana API.
+This requires a Kibana endpoint configuration. You should have configured the
+endpoint earlier when you
+<<{beatname_lc}-configuration,configured {beatname_uc}>>. If you didn't,
+configure it now.
+
+Make sure Kibana is running before you perform this step. If you are accessing a
+secured Kibana instance, make sure you've configured credentials as described in
+<<{beatname_lc}-configuration>>.
+
+To set up the Kibana dashboards for {beatname_uc}, use the appropriate command
+for your system.
+
+ifdef::allplatforms[]
+
+ifeval::["{requires-sudo}"=="yes"]
+
+include::../../libbeat/docs/shared-note-sudo.asciidoc[]
+
+endif::[]
+
+*deb and rpm:*
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+{beatname_lc} setup --dashboards
+----------------------------------------------------------------------
+
+
+*mac:*
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+./{beatname_lc} setup --dashboards
+----------------------------------------------------------------------
+
+
+ifeval::["{beatname_lc}"!="auditbeat"]
+
+*docker:*
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+docker run {dockerimage} setup --dashboards
+----------------------------------------------------------------------
+
+endif::[]
+
+*win:*
+
+endif::allplatforms[]
+
+Open a PowerShell prompt as an Administrator (right-click the PowerShell icon
+and select *Run As Administrator*). If you are running Windows XP, you may need
+to download and install PowerShell.
+
+From the PowerShell prompt, change to the directory where you installed {beatname_uc},
+and run:
+
```

Comment:
Something is very strange regarding including setup cmd here, I see Power shell instructions on a non-windows machine:


![screen shot 2018-01-29 at 14 42 14](https://user-images.githubusercontent.com/5555349/35513218-9f9282dc-0502-11e8-8314-e6f13ec859a8.png)

Reply:
i think thats fixed then it goes live